proto/arkeo/claim/query.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package arkeonetwork.arkeo.claim;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "arkeo/claim/params.proto";
+
+option go_package = "github.com/arkeonetwork/arkeo/x/claim/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  
+  // Parameters queries the parameters of the module.
+  rpc Params (QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get = "/arkeonetwork/arkeo/claim/params";
+  
+  }
+  
+  // Queries a list of ClaimRecord items.
+  rpc ClaimRecord (QueryClaimRecordRequest) returns (QueryClaimRecordResponse) {
+    option (google.api.http).get = "/arkeonetwork/arkeo/claim/claim_record/{address}";
+  
+  }
+}
+// QueryParamsRequest is request type for the Query/Params RPC method.
+message QueryParamsRequest {}
+
+// QueryParamsResponse is response type for the Query/Params RPC method.
+message QueryParamsResponse {
+  
+  // params holds all the parameters of this module.
+  Params params = 1 [(gogoproto.nullable) = false];
+}
+
+message QueryClaimRecordRequest {
+  string address = 1;
+}
+
+message QueryClaimRecordResponse {
+  string claimRecord = 1;
```

Comment:
FYI: https://developers.google.com/protocol-buffers/docs/style
google encourage field to use snake case